src/extensions/default/analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers.Common/AnalyzerExtensions.cs

```diff
@@ -4,43 +4,20 @@
 using System;
 using System.Threading;
 using Microsoft.CodeAnalysis;
-using Microsoft.CodeAnalysis.CSharp.Syntax;
 using Microsoft.CodeAnalysis.Diagnostics;
 using Microsoft.CodeAnalysis.Operations;
 
-namespace Microsoft.DotNet.UpgradeAssistant.Extensions.Default.CSharp.Analyzers
+namespace Microsoft.DotNet.UpgradeAssistant.Extensions.Default
 {
-    internal static class DiagnosticAnalyzerExtensions
+    public static class AnalyzerExtensions
```

Comment:
I updated all the extension classes to be public so that they can be shared between the analyzer project and the code fix provider project. This particular one is only used by analyzers, so we could move it back to the analyzer project and make it internal.